Sources/TokamakCore/Tokens/Color.swift

```diff
@@ -15,38 +15,178 @@
 //  Created by Max Desiatov on 16/10/2018.
 //
 
-public struct Color: Hashable, Equatable {
-  public static func == (lhs: Self, rhs: Self) -> Bool {
-    var lightEnv = EnvironmentValues()
-    lightEnv.colorScheme = .light
-    var darkEnv = EnvironmentValues()
-    darkEnv.colorScheme = .dark
-    return lhs._evaluate(lightEnv) == rhs._evaluate(lightEnv) &&
-      lhs._evaluate(darkEnv) == rhs._evaluate(darkEnv)
+/// Override `TokamakCore`'s default `Color` resolvers with a Renderer-specific one.
+/// You can override a specific color box (such as `_SystemColorBox`, or all boxes with `AnyColorBox`.
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Tokens/Color.swift;
  Line: 19;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 102 characters (`line_length`)